ComSemApp/templates/ComSemApp/teacher/attempt_request_info.html

```diff
@@ -0,0 +1,105 @@
+{% load static %}
+<div class="row h-100">
+    <div class="col-12 h-100">
+        <div class="card-box">
+            <div class="card">
+                <div class='card-body'>
+                    <!--card for displaying results-->
+                    <b>Student</b><br>
+                    {{attempt.student}} <br>
+                    <b>Course</b><br>
+                    {{ course.session }}<br>
+                    <b>Correct Expression</b><br> 
+                    {{attempt.expression.reformulation_text}} <br>
+                    <b>Problem Expression</b><br> 
+                    {{attempt.expression.expression}} <br>
+                    <b>Graded Expression</b> (hover for details about each error)<br> 
+                    <mark><span title="There could be errors shown here">{{attempt.transcription}}</span></mark><br>
+                    <b>Audio Recording</b><br>
+                    <audio controls title="Reformulation" style="width: 200px" class="previous_attempt_audio" src="{{ attempt.audio.url }}"></audio><br>
+                    <b>Fluency Score</b><br>
+                    {{attempt.wpm}}<br>
+                    <b>Correctness Score</b><br>
+                    {% if review.original_score%}
+                        {{ review.original_score}}
+                    {% else %}
+                        {{attempt.correct}}
+                    {% endif %}
+                    <br>
+                    <b>New Correctness Score <input type="number" min="0" max="100" style="width:85px;" id="myPercent" {% if review.original_score %}value="{{ attempt.correct }}"{% endif %}></b>
```

Comment:
style tag should be bootstrap class

Reply:
I am unsure what I would change to bootstrap?

Reply:
Can't set specific width with bootstrap. Smallest sizing option in bootstrap is too long